src/test/rustdoc-js/generics.js

```diff
@@ -3,7 +3,6 @@
 const QUERY = [
   '"R<P>"',
   '"P"',
-  'P',
```

Comment:
It seems odd that you’re removing the non-quoted search, but not removing the quoted search.

Reply:
I updated the test to take the update into account.